mava/utils/total_timestep_checker.py

```diff
@@ -47,3 +47,33 @@ def check_total_timesteps(config: DictConfig) -> DictConfig:
             + f"{Style.RESET_ALL}"
         )
     return config
+
+
+def sebulba_check_total_timesteps(config: DictConfig) -> DictConfig:
+    """Check if total_timesteps is set, if not, set it based on the other parameters"""
+
+    if config.system.total_timesteps is None:
```

Comment:
since the is a lot in common we can use a single function for both and create two variables like `n_devices `and `update_batch_size` and the rest is the same as the anakin function:
```
 if config.arch.architecture_name =="anakin":
        n_devices = len(jax.devices())
        update_batch_size = config.system.update_batch_size
 else:
        n_devices = len(config.arch.executor_device_ids)
        update_batch_size = config.arch.n_threads_per_executor
   
```